src/locales/messages.ts

```diff
@@ -2071,6 +2262,11 @@ export default defineMessages({
     description: 'Perspective values',
     id: 'PerspectiveValues',
   },
+  Rate: {
+    defaultMessage: 'EN Rate',
+    description: 'Rate',
+    id: 'Rate',
+  },
```

Comment:
This is a dup of CostModelsTagRateTableRate

Thinking CostModelsTagRateTableRate should be replaced with this one.